net-mail/poppassd_ceti/poppassd_ceti-1.8.7.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pam readme.gentoo-r1
+
+# Long ago it was just "poppassd", but upstream now seems to have
+# settled on "poppassd-ceti" (instead of "poppassd_ceti" or no suffix).
+MY_PN="poppassd-ceti"
+MY_P="${MY_PN}-${PV}"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Password change daemon with PAM support"
+HOMEPAGE="https://github.com/kravietz/poppassd-ceti"
+SRC_URI="https://github.com/kravietz/${MY_PN}/releases/download/v${PV}/${MY_P}.tar.xz"
+
+# Strictly speaking the "or later version" clarification was only
+# added upstream after 1.8.7:
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-libs/pam"
+RDEPEND="${DEPEND}"
+
+FORCE_PRINT_ELOG=1  # possibly remove in the next bump
+DISABLE_AUTOFORMATTING=1
+DOC_CONTENTS="poppassd is installed, but has to be run as root to work.
+
+Most commonly a front end would require sys-apps/xinetd and
```

Comment:
Could you reformat this to have consistent line width? Right now it seems to be wrapped semi-randomly.

Reply:
done